Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -301,40 +346,6 @@ public void StopClimbing()
 		}
 	}
 
-	protected void CheckGroundSlope()
```

Comment:
Removing this brings back the problem of sliding up Unity terrain. It's still happens and I think we need a solution before we remove this.